styleguide/derek/solutions/solutions.ejs

```diff
@@ -41,10 +41,18 @@
 <%- partial('../_partials/solutions/tabs.ejs') %>
 <%- partial('../layouts/lists.ejs') %>
 <%- partial('../layouts/announcements.ejs') %>
-
-
 <%- partial('../_partials/solutions/blog-feed.ejs') %>
 
+<!-- Rs Tabs -->
+<div class="container">
+  <h4>Rs tabs</h4>
+  <div class="row">
+    <div class="col-xs-12">
+      <%- partial('../_partials/solutions/tabs.ejs') %>
+    </div>
+  </div>
+</div>
```

Comment:
Can you throw all this wrapping code into a `../layouts/tabs.ejs` file like we've done with some of the other patterns? This will help us keep the solutions page relatively clutter free.